docs/docs/user-permissions/access-control.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Access Control in the API
+sidebar_position: 2
+---
+
+:::note
+
+The term **operation** stands for the locations in which COMET DXP invokes permission checks:
+
+-   Queries/Mutations in GraphQL-resolvers
+-   Routes in REST-controllers
+
+Normally you want to decorate the methods of these classes, however, decorating the whole class is also possible.
+:::
+
+After activating the module, COMET DXP checks every operation for the required permissions and scopes. Therefore it is necessary to decorate the operations to let the system know what to check. COMET DXP then checks if the current user possesses the permission defined in the decorator.
+
+Additionally, the scope of the data in operation will be checked against the scope of the users. To achieve this, the system has to know the scope of the data that is being handled right now.
+
+:::note
+You might also want to check the permissions on field resolvers. Therefore you have to add `guards` to `fieldResolverEnhancers` in the configuration of the GraphQL-module. Please be aware that field resolvers are only checked for permissions but not for scopes.
+:::
+
+## Permission check
+
+**@RequiredPermission**
+
+This decorator is mandatory for all operations. The first parameter of type `string | string[] | "disablePermissionCheck"` configures which permission is necessary to access the decorated operation.
+
+The core of COMET DXP already defines a list of permissions (e.g. `pageTree`, `dam`, `cronJobs`, `userPermissions`). Permissions are defined as plain strings, in the most basic case they represent the main items of the menu bar in the administration panel.
```

Comment:
We possibly should provide more information when to use a Comet-internal permission in projects, e.g., pageTree for pages

Reply:
We can add this later if required.